packages/regl-worldview/src/Worldview.js

```diff
@@ -206,7 +213,7 @@ export class WorldviewBase extends React.Component<BaseProps, State> {
       .readHitmap(canvasX, canvasY)
       .then((objectId) => {
         if (worldviewHandler) {
-          handleWorldviewMouseInteraction(objectId, ray, e, worldviewHandler);
+          handleWorldviewMouseInteraction(worldviewContext.getDrawPropByHitmapId(objectId), ray, e, worldviewHandler);
```

Comment:
We can declare `const object = worldviewContext.getDrawPropByHitmapId(objectId)` here and use directly for `worldviewContext.callComponentHandlers(object)`. Also need to update Command's `handleMouseEvent`